Client/Frontend/Contextual Hint/ContextualHintViewModel.swift

```diff
@@ -121,7 +121,7 @@ class ContextualHintViewModel {
         case .toolbarLocation:
             switch arrowDirection {
             case .up:
-                return CFRStrings.Toolbar.SearchBarPlacementForExistingUsers
```

Comment:
I'm assuming the deletion of `SearchBarPlacementForExistingUsers` will be in [this ticket](https://mozilla-hub.atlassian.net/browse/FXIOS-4628).

Reply:
I generally wouldn't make that assumption, especially if you're removing the code that point to that string. I can keep it in mind, though, and can remove it in that ticket.